documents/charter.md

```diff
@@ -1,23 +1,24 @@
-Video traffic is already 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
+Video traffic is 70% of the overall traffic volume on the Internet and is expected to grow to 80% by 2028.
 Across developed and emerging markets video traffic forms 50-80% of traffic volume on mobile networks.
 New formats like short form videos have seen tremendous growth in recent years.
 These growth trends are likely to increase with new populations coming online on mobile-first markets.
 
-Mobile network operators continuously invest in network resources, including deployment of new generations or new bands of spectrum.
-Since spectrum is a limited and expensive resource, operators often make use of flow-based traffic handling such as shaping of video traffic, especially when the network is highly loaded.
-Operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+While mobile network operators continuously invest in new network capacity, including deployment of new generations or new bands of spectrum, these efforts are not keeping pace with growth in demand. These network operators have found it faster and less expensive to invest in shaping of video traffic on a per-flow basis, which negatively affects video stream quality. Because that traffic is encrypted, network operators cannot explicitly measure the degradation that shaping causes to end user quality of experience (QoE), making this approach open loop. 
+
```

Comment:
> This and the para that follows seems to be directed to a use case using cabled infrastructure, where this might be less of a concern?

Maybe. In my experience the mechanisms to add capacity in wired and wireless networks are quite similar. Since in the US the wireline networks have been barred from app or protocol-specific shaping, they have perhaps invested in more peak hour capacity at the edge.